meta-mender-tegra/recipes-mender/tegra-state-scripts/files/redundant-boot-install-script-uboot

```diff
@@ -50,21 +55,31 @@ mount --bind /dev "${mnt}/dev"
 # Mount an overlay for /var/run so the /var/lib/nv_boot_control.conf symlink becomes writable
 mount -t tmpfs tmpfs ${mnt}/run
 
-# Run the update engine in the context of the just-installed rootfs
-# to ensure that the TNSPEC in the config file it uses matches the
-# TNSPEC in the update payload. But first we have to set up the
-# configuration file with the TNSPEC.
-if [ -L "${mnt}/etc/nv_boot_control.conf" -a -x "${mnt}/usr/bin/setup-nv-boot-control" ]; then
-    mkdir -p "${mnt}/run/tegra-nv-bootctrl"
-    if ! chroot "${mnt}" /usr/bin/setup-nv-boot-control; then
-	echo "ERR: could not initialize nv_boot_control.conf in new rootfs" >&2
-    fi
+if [ -f "${mnt}/etc/nv_boot_control.conf" ]; then
```

Comment:
@anishmonachan7 is the issue that you are updating from a rootfs which doesn't have the `nv_boot_control.conf` symlink setup [here](https://github.com/mendersoftware/meta-mender-community/blob/dunfell/meta-mender-tegra/recipes-bsp/tegra-binaries/tegra-nv-boot-control-config_%25.bbappend#L2)?

I think we'll want to keep the logic which handles the symlink case so we could still support delta updates, however maybe we should have logic which can handle the case `${mnt}/etc/nv_boot_control.conf` exists, remount read write, and add a print which states delta updates aren't supported for this case.